sonar-batch-protocol/src/main/protobuf/batch_report.proto

```diff
@@ -144,11 +145,21 @@ message Duplication {
   repeated Duplicate duplicate = 2;
 }
 
+// Used for cross project duplication
```

Comment:
In fact I've tried to remove it but it's used by PersistDuplicationsStep -> As you'll work on this class, I think it's more simpler for you to do the change.